1.data_preprocessing/2.format_bulk.ipynb

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.        overlapping_rows = expr_df.index.intersection(metadata_df.index)
</pre><p>you also only want to subset to samples that have a non-NA value in the column "ran_in_way_pipeline"</p><p> </p><p>unless you are interested in looking at effects of different sample properties (neoadjuvant chemotherapy, sequencing depth, etc.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NDavidsonLab/BuDDI_HGSC_schildkraut_analysis/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>